packages/tldraw/src/lib/shapes/text/TextShapeUtil.tsx

```diff
@@ -70,31 +70,30 @@ export class TextShapeUtil extends ShapeUtil<TLTextShape> {
 		} = shape
 
 		const { width, height } = this.getMinDimensions(shape)
-		const isSelected = shape.id === this.editor.getOnlySelectedShape()?.id
+		const isSelected = shape.id === this.editor.getOnlySelectedShapeId()
+		const theme = useDefaultColorTheme()
 
 		return (
-			<HTMLContainer id={shape.id}>
-				<TextLabel
-					id={id}
-					classNamePrefix="tl-text-shape"
-					type="text"
-					font={font}
-					fontSize={FONT_SIZES[size]}
-					lineHeight={TEXT_PROPS.lineHeight}
-					align={align}
-					verticalAlign="middle"
-					text={text}
-					labelColor={color}
-					isSelected={isSelected}
-					textWidth={width}
-					textHeight={height}
-					style={{
-						transform: `scale(${scale})`,
-						transformOrigin: 'top left',
-					}}
-					wrap
-				/>
-			</HTMLContainer>
+			<TextLabel
+				id={id}
+				classNamePrefix="tl-text-shape"
+				type="text"
+				font={font}
+				fontSize={FONT_SIZES[size]}
+				lineHeight={TEXT_PROPS.lineHeight}
+				align={align}
+				verticalAlign="middle"
+				text={text}
+				labelColor={theme[color].solid}
+				isSelected={isSelected}
+				textWidth={width}
+				textHeight={height}
+				style={{
+					transform: `scale(${scale})`,
+					transformOrigin: 'top left',
+				}}
+				wrap
+			/>
```

Comment:
No need for the extra element here (I don't think)

Reply:
cc @SomeHats I'm not sure - this is pretty new and I think it might be necessary for SVG export - is that right @SomeHats ?